src/api/cow/index.ts

```diff
@@ -103,19 +103,19 @@ export class CowApi {
   }
 
   get API_BASE_URL() {
-    return getGnosisProtocolUrl(this.context.isDevEnvironment)
+    return getGnosisProtocolUrl(this.context.env === 'staging')
```

Comment:
This function signature looks weird now.

A refactor suggestion is to pass the `env` as parameter, and on `getGnosisProtocolUrl`, use a `switch`.
When all types are covered you don't need to add a `default` clause and TS will be happy.

Something like:

```ts
switch(env) {
 case 'staging': return 'staging url....'
 case 'prod': return 'prod url..'
}

Reply:
`getGnosisProtocolUrl` -> `getCowProtocolUrl` ? does it make sense to do it in this PR?